### DISCUSSION TITLE
MQTT Data Bridge: How to subscribe to multiple topics?

### DISCUSSION THREAD

**Discussion by matthewGEA** [emqx/emqx#10424] [General]
Similar to [this discussion](https://github.com/emqx/emqx/discussions/10241) on publishing multiple topics via a single bridge:

How can one configure EMQX to subscribe to multiple topics via a single bridge?

Version 5.0.12 included a refactoring of MQTT bridge config. The older version config file created from v5.0.11 or earlier will be converted to according to the new schema. Please note though, the request body of /bridges API to configure MQTT bridge is changed in a incompatible way.

Prior to 5.0.12, it was possible to use this syntax:
```
bridges.mqtt.bridge1.connector = connectorID
bridges.mqtt.bridge1.direction = ingress
bridges.mqtt.bridge1.remote_topic = topic1/#

bridges.mqtt.bridge2.connector = connectorID
bridges.mqtt.bridge2.direction = ingress
bridges.mqtt.bridge2.remote_topic = topic2/#
```

**Comment by savonarola** [comment]
Hello!
As far as I understand, the only way to achieve this currently is to create several ingress bridges. 

We will also consider possible ways to optimize this.
